pkg/render/logstorage/linseed/linseed_test.go

```diff
@@ -395,6 +400,37 @@ func expectedContainers() []corev1.Container {
 					Value:     "https://tigera-secure-es-http.tigera-elasticsearch.svc:9200",
 					ValueFrom: nil,
 				},
+				{
+					Name:      "LINSEED_ELASTIC_USERNAME",
+					Value:     "elastic",
+					ValueFrom: nil,
+				},
+				{
+					Name:  "LINSEED_ELASTIC_PASSWORD",
+					Value: "",
+					ValueFrom: &corev1.EnvVarSource{
+						FieldRef:         nil,
```

Comment:
All of these can just be omitted - no need to explicitly set them to their default value, just makes the code more verbose.